src/datumaro/plugins/data_formats/yolo/base.py

```diff
@@ -200,13 +200,13 @@ def _get(self, item_id: str, subset_name: str) -> Optional[DatasetItem]:
 
         return item
 
-    @staticmethod
-    def _parse_field(value: str, cls: Type[T], field_name: str) -> T:
+    @classmethod
+    def _parse_field(cls, value: str, type: Type[T], field_name: str) -> T:
```

Comment:
`type` is a built-in keyword of Python. Please use another one.

Reply:
fixed at https://github.com/openvinotoolkit/datumaro/pull/1044/commits/7f4aabc493568ea500f7d65699238810904724db